File: Modul-5/Soal-1/PRAK501-2310817110013-Randy_Febrian.py
def MaxBilangan(i, j, k, l):
    if i > j and i > k and i > l:
        return(i)
    elif j > i and j > k and j > l:
        return(j)
    elif k > i and k > j and k > l:
        return(k)
    elif l > i and l > j and l > k:
        return(l)
    
a, b, c, d = map(int, input().split())
hasil = MaxBilangan(a, b, c, d)
print("%d" % (hasil))

File: Modul-5/Soal-1/PRAK501-2310817110013-Randy_Febrian.c
#include <stdio.h> 
 
//Buatlah Function Disini 
int MaxBilangan(int i, int j, int k, int l){
    if(i > j && i > k && i > l){
        return(i);
    }
    else if(j > i && j > k && j > l){
        return(j);
    }
    else if(k > i && k > j && k > l){
        return(k);
    }
    else if(l > i && l > j && l > k){
        return(l);
    }
}

int main() {  	
    int a, b, c, d;      	
    scanf("%d %d %d %d", &a, &b, &c, &d); 
    int hasil = MaxBilangan(a, b, c, d); 
    printf("%d", hasil);    
    return 0; 
} 
